### ISSUE TITLE
Add HTML/SCSS

### ISSUE THREAD

**Issue by PhantomRex123** [PhantomRex123/Arduino-Smart-House#1]
We have to make the HTML website, in the website/ directory, i'll work on it, any contribution would be loved!

**Comment by Akshata-Gunapache** [comment]
I would love to contribute to this issue. Could you please give further details on the issue?

**Comment by PhantomRex123** [comment]
> I would love to contribute to this issue. Could you please give further details on the issue?

Sure. You can merge this into the webApp branch , and then we'll migrate it in the codebase. We'll use SCSS and Typescript, if you can't do typescript section, I'll do the typescript part ! 

About the design part. We need dynamic components, AKA if i add a a new element the positioning won't have any issues - You can use a CSS libary like boot-strap. We need animated light components, When they are clicked they go to blue(on) else they go to gray(off). We have some stat components, where we share Sensor running time, room temp, etc... 

A simple login page - Dashboard - Settings Page Are all we need!  Color scheme is your choice, It really needs to be mobile responsive, nothing else.

Very nice designs:
https://dribbble.com/shots/14953938-Smart-home-app-concept - Sleek , elegent!
https://dribbble.com/shots/16237200-Smart-Home-App-Concept
https://dribbble.com/shots/16224440-Smart-Home-App
https://dribbble.com/shots/15653918-Smart-Home-App
https://dribbble.com/shots/15724326-Smart-Home-App
https://dribbble.com/shots/8899139-Smart-Home
https://dribbble.com/shots/15848937-SmartHome-App-Platform - Fun! i love this one :)

**Comment by Akshata-Gunapache** [comment]
@PhantomRex123 Yes I got it. I will soon start working on this and keep updating you with the progress. Thank You

**Comment by PhantomRex123** [comment]
ok - i added node-sass and autoprefixer.

**Comment by PhantomRex123** [comment]
Any updates?

**Comment by Akshata-Gunapache** [comment]
> 

Not yet...I am working on it . Do you want to add a logo to the website too?...Is it going to be a website for buying the appliances or controlling the smart home devices?

**Comment by Akshata-Gunapache** [comment]
> > I would love to contribute to this issue. Could you please give further details on the issue?
> 
> Sure. You can merge this into the webApp branch , and then we'll migrate it in the codebase. We'll use SCSS and Typescript, if you can't do typescript section, I'll do the typescript part !
> 
> About the design part. We need dynamic components, AKA if i add a a new element the positioning won't have any issues - You can use a CSS libary like boot-strap. We need animated light components, When they are clicked they go to blue(on) else they go to gray(off). We have some stat components, where we share Sensor running time, room temp, etc...
> 
> A simple login page - Dashboard - Settings Page Are all we need! Color scheme is your choice, It really needs to be mobile responsive, nothing else.
> 
> Very nice designs: https://dribbble.com/shots/14953938-Smart-home-app-concept - Sleek , elegent! https://dribbble.com/shots/16237200-Smart-Home-App-Concept https://dribbble.com/shots/16224440-Smart-Home-App https://dribbble.com/shots/15653918-Smart-Home-App https://dribbble.com/shots/15724326-Smart-Home-App https://dribbble.com/shots/8899139-Smart-Home https://dribbble.com/shots/15848937-SmartHome-App-Platform - Fun! i love this one :) The last one is very nice , since we also are working with a camera module that would be very useful.

These are designs for apps ...for websites we need to work on it differently.

**Comment by PhantomRex123** [comment]
> > 
> 
> Not yet...I am working on it . Do you want to add a logo to the website too?...Is it going to be a website for buying the appliances or controlling the smart home devices?

No, we don't even have a logo - no need for evtra clutter. also its for controlling the smart home devices.

**Comment by Akshata-Gunapache** [comment]
Could u please give a basic layout of how the website should look?

**Comment by Akshata-Gunapache** [comment]
Should it be something like this-

![IMG_20220317_081833](https://user-images.githubusercontent.com/96786500/158726630-012f97d0-36fb-41d9-876c-7f67730942d4.jpg)
(It was a sample idea)

**Comment by PhantomRex123** [comment]
> Should it be something like this-
> 
> ![IMG_20220317_081833](https://user-images.githubusercontent.com/96786500/158726630-012f97d0-36fb-41d9-876c-7f67730942d4.jpg) (It was a sample idea)

Yes, its awesome.

**Comment by Akshata-Gunapache** [comment]
@PhantomRex123 Sorry for the delay . I am busy with my exams and college . I have just made a grid layout of the webpage . Do I push it here?
![Screenshot 2022-04-10 104219](https://user-images.githubusercontent.com/96786500/162602440-48c11233-86c1-4e1d-a8da-21855800f3f4.png)
It looks like this . You can make changes to it according to your preferences.

**Comment by PhantomRex123** [comment]
> @PhantomRex123 Sorry for the delay . I am busy with my exams and college . I have just made a grid layout of the webpage . Do I push it here? ![Screenshot 2022-04-10 104219](https://user-images.githubusercontent.com/96786500/162602440-48c11233-86c1-4e1d-a8da-21855800f3f4.png) It looks like this . You can make changes to it according to your preferences.

yes
